bin/dotd_menu

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env ruby
+
+# This script attempts to automate and guide the Developer Of The Day (DOTD) through our deploy
+# processes. As much of this process is automated, the script is now a menu driven script rather
+# than a linear workflow driven script.
+
+# TODO(asher): Rename this to `bin/dotd` (removing the linear workflow driven script) when
+# appropriate.
+
+require 'cgi'
+require 'io/console'
+require 'logger'
+require 'tmpdir'
+require_relative '../lib/cdo/cdo_cli'
+require_relative '../lib/cdo/developers_topic'
+require_relative '../lib/cdo/infra_test_topic'
+require_relative '../lib/cdo/infra_production_topic'
+require_relative '../lib/cdo/github'
+require_relative '../lib/cdo/honeybadger'
+include CdoCli
+
+LOG_FILE = "#{Dir.tmpdir}/dotd.log".freeze
+@logger = Logger.new(LOG_FILE, 'daily')
+
+@logger.formatter = proc do |_serverity, time, _progname, msg|
+  "[#{time.strftime('%I:%M %p')}]: #{msg}\n"
+end
+
+# Checks that CDO.github_access_token and CDO.slack_token are defined. If not,
+# exits the program.
+def check_for_cdo_keys
+  return if CDO.github_access_token && CDO.slack_token && CDO.honeybadger_api_token
+
+  puts <<-EOS.unindent
+
+    This script requires CDO.github_access_token, CDO.slack_token, and CDO.honeybadger_api_token.
+
+    Create your API tokens from these pages:
+
+      https://github.com/settings/tokens ('public_repo' permission)
+      https://api.slack.com/custom-integrations/legacy-tokens
+      https://app.honeybadger.io/users/edit#authentication
+
+    Please add them to your locals.yml and rerun the script.
+
+  EOS
+  exit
+end
+
+# Prints the script intro to the console.
+def puts_script_intro
+  # TODO(asher): Update this appropriately.
```

Comment:
Given how recently (or not) the Known Issues has been touched, I wanted to discuss other options for it (perhaps something managed by DOTD script). If nothing else, the TODO was to remind me of that discussion.

That said, I am taking your advice and removing the TODO.